src/TaylorInversion.jl

```diff
@@ -111,13 +102,14 @@ function further_substitution(it::InverseTaylor{N}, subbed::Num) where {N}
     return subbed
 end
 
-function create_expressions(n::Int)
-    it = InverseTaylor{n}()
+@generated function (::TaylorInverter{N})(var"ˍ₋arg1") where {N}
```

Comment:
GIven the fact that the `@generated` macro induces a behaviour that might surprise people, I would quite appreciate some comments being added to this function. In particular:
1. A comment on the name of the argument (`var"_-arg1` not sure what the best place is for this)
2. A comment that the return value of this function as read "naively" is _not_ what the function will return, but a string that will be evaluated when the code is lowered. That is, if I understand correctly what happens with a `@generated` function, kind of proving that it would be wise to document this (briefly, I understand we should not rewrite the Metaprogramming page of the Julia Docs.
3. Providing a docstring for this function that indicates the behaviour of the _generated_ function. One of the (many) joys of Julia is it's innate readability, but with a generated function, this is somewhat hampered, for instance by the obscure argument name.

Reply:
> * A comment on the name of the argument (`var"_-arg1`, not sure what the best place would be for this comment)

It's the (Symbolics.jl) mangled name of the symbolic argument which is defined by this package. The argument name is the same as the argument name of the function that you eval'd

> 2\. A comment that the return value of this function as read "naively" is _not_ what the function will return, but a string that will be evaluated when the code is lowered.

The generator returns an AST.

> obscure argument name.

The argument name is defined by this package when choosing the symbolic Num - if you change that, you get whatever symbol you want.